src/energyplus/Test/EMS_GTest.cpp

```diff
@@ -2090,8 +2090,8 @@ TEST_F(EnergyPlusFixture, ForwardTranslatorActuator_exampleModel_Electric_EMS) {
     EnergyManagementSystemActuator electricActuator1(electricEquipment[1], ComponentType, ControlType);  // , space4.get());
   } else {
     // 0 is on a SpaceType, 1 is on a Space
-    EnergyManagementSystemActuator electricActuator0(electricEquipment[0], ComponentType, ControlType);  // , space4.get());
     EnergyManagementSystemActuator electricActuator1(electricEquipment[1], ComponentType, ControlType);
+    EnergyManagementSystemActuator electricActuator0(electricEquipment[0], ComponentType, ControlType);  // , space4.get());
```

Comment:
Make sure the EMS actuators are instantiated in the same order for this test to pass. The warning will be issued for the one attached a Space Type (the other is on a space), so we make sure they are always created in the same fashion